pkg/master/client_ca_hook.go

```diff
@@ -22,13 +22,13 @@ import (
 	"reflect"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
pull this and related into a separate pull that we can easily merge (much smaller and less risk)

Reply:
makes perfect sense

Reply:
ref: #70520